### ISSUE TITLE
How can I know what the train is for?

### ISSUE THREAD

**Issue by 745275633** [Yousei9/Factorio-LogisticTrainNetwork#37]
I have a stop for provided and requested, but when a train go to send something, it will also get something.

**Comment by 745275633** [comment]
Or train need get something, but it get other things.

**Comment by Yousei9** [comment]
Sounds like your stations are not set to read loading list and train inventory.

Please check the forum thread, especially my [opening post](https://forums.factorio.com/viewtopic.php?f=94&t=36976) for station setups using circuit network with filter inserters.

Stations capable of loading AND unloading can be found in the demo map or [here](https://forums.factorio.com/viewtopic.php?f=94&t=36976&start=680#p279647)

**Comment by 745275633** [comment]
sorry, I think I know.

**Comment by 745275633** [comment]
Look like I do something wrong, let me try again

**Comment by 745275633** [comment]
Maybe I do something wrong, but I think it did not show at output signals before I restarted the game.

**Comment by 745275633** [comment]
Yes, it's works good, but I found another problem, I can't get what in train.

**Comment by 745275633** [comment]
A train with lots of iron plate in stop, and connect stop(the biggest part) and lamp.When I set `Iron plate = 0` ,the lamp is on
